### ISSUE TITLE
incoming message problem

### ISSUE THREAD

**Issue by iddq** [ge0rg/aprsdroid#217]
if you get many messages from someone then later you cannot see the newer messages from this person however you get the messages and you can see the messages in the debug window.

to reproduce this bug do the following:
send a message "d" to ANSRVR
you will get lot of messages ANSRVR
later send a message "help"
you will get the answer but it will not appear in the message window.

**Comment by ge0rg** [comment]
Thank you for the report. There is probably a message limit gone wrong. I'll try to reproduce and change the limits for the next version.

**Comment by ge0rg** [comment]
I've checked the source and there is no limitation on the number of messages. 

Also I can't reproduce with messages to ANSRVR. Whenever you send a new message, the screen will scroll to the bottom, and then new messages will appear there as well.

Could you re-check with the latest version of APRSdroid?

**Comment by iddq** [comment]
What is the latest version?

**Comment by iddq** [comment]
I can still reproduce this issue.

**Comment by debugrr** [comment]
Unable to reproduce on 1.6.0e

**Comment by iddq** [comment]
so you get the help message at the end of the test, right?

**Comment by debugrr** [comment]
Yes, I did it 2x... hope my local APRS users don't kill me for spamming :D

**Comment by iddq** [comment]
I'm doing this with APRS-IS
